### PR TITLE
update bioformats to 7.1.0

### PR DESCRIPTION
**Pull request by sameeul**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+[build-system]
+requires = [
+  "setuptools", 
+  "wheel",
+  "imagecodecs>=2021.2.26",
```
Comment (jfennick) [inline pyproject.toml:5]: Upto you, but I would probably add a comment here saying that the runtime dependencies have been added as build-time dependencies, simply due to the fact that [src/bfio/__init__.py exports some aliases](https://github.com/PolusAI/bfio/pull/68/files#diff-56f4b8a4749625a073696f969c033711fd761ffcb170261dc3bbc368719c72d6R29), which causes cascading imports which effectively requires adding all runtime dependencies at build-time.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,6 @@
 __pycache__
 .vscode
-.venv
+venv
```
Comment (jfennick) [inline .gitignore:3]: Just checking that you intended to remove the "."  If this is correct, then LGTM.

**Review by jfennick — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (sameeul) [reply pyproject.toml:5]: will do that!

File: .gitignore (hunk 2 above)
Comment (sameeul) [reply .gitignore:3]: intentional change.

**Pull request merged**